app/services/bulk_upload/lettings/csv_parser.rb

```diff
@@ -52,6 +52,7 @@ def normalised_string
 
     @normalised_string = File.read(path, encoding: "bom|utf-8")
```

Comment:
Do you know what the invalid byte sequence was? This sounds like a BOM issue (we've hit this a few times on the old service) but looks like this line should handle it without any additional scrubbing.

Reply:
i see it as `<92>` so i'm guessing it's some kind of apostrophe or single smart quote. the box recognised the file as
```sh
file THE_FILE
/home/vcap/tmp/20230223-233-240fys: Non-ISO extended-ASCII text, with very long lines, with CRLF, LF line terminators
```

Reply:
Sorry there are also more sequencess:
- `<A0>` - not sure on this one?
- `<93>` - start smart double quote
- `<94>` - end smart double quote

Reply:
actually let me get back to you it's possible it maybe something else